lib/engine/game/g_18_zoo.rb

```diff
@@ -546,6 +546,42 @@ def self.title
         '18ZOO'
       end
 
+      def purchasable_companies(entity = nil)
+        entity ||= @round.current_operator
+        return [] unless entity && (entity.corporation? || entity.player?)
```

Comment:
don't use unless with multiple conditoions change it to an if

Reply:
done in 955e5710